flux-pipe/plotting/plot_scalability.py

```diff
@@ -5,8 +5,9 @@
 import pandas as pd
 # import statsmodels.api as sm
 
-datdir = "/Users/cgilbert/vscode/Fluxon-Scripts-Gilly/"
-file_path = os.path.join(datdir, "fluxon/pfss_time.txt")
+datdir = "/Users/cgilbert/vscode/fluxon-data/"
```

Comment:
Local directory

Reply:
to be fixed later, this is also a dev plot now